index.js

```diff
@@ -2,7 +2,8 @@ const regex = /\n\n<!-- probot = (.*) -->/
 
 module.exports = (context, issue = null) => {
   const github = context.github
-  const prefix = context.payload.installation.id
+  let prefix = process.env.APP_ID
```

Comment:
```js
const prefix = context.payload.installation && context.payload.installation.id ? process.env.APP_ID
```

Reply:
The proposal is to use the app id going forward and fall back to the app id, if missing; so the initialization above will need to be reversed, but i get that idea 👍 

But the reason to keep both was so that in the `get`, it can try to load data from the old or new prefix, and keep the change backward compatible as per comments from Brandon in slack.

> re breaking change, my only problem with that approach is that there’s not a good way for users to migrate to the new version. So I think it’s easy enough to include check for old vs new keys at read time `data[prefix] || data[oldPrefix]` - Brandon

so keeping them both in play, leads to the following associated change in the `get`:

`const data = json[prefix] || json[oldPrefix]`